distribution/docs/src/main/resources/content/_sources/solr-catalog-provider.adoc

```diff
@@ -11,153 +11,88 @@
-* External installation allows isolation of data storage.
-* Requires additional protection to be secured.
-* Not scalable.
-* External installation has network overhead
-
-.Installing HTTP Solr
-
-By default, the ${ddf-solr} application installs a Solr server inside the same container as {branding}.
-To install an External Solr server, refer to https://cwiki.apache.org/confluence/display/solr/Apache+Solr+Reference+Guide[Apache Solr Reference Guide].
-
-.Configuring HTTP Solr
+* Enable TLS. By default, Solr does not use transport layer security (TLS) because ${branding}
+and Solr are running on the same host. To enable TLS:
 
 . Edit the `${home_directory}/etc/system.properties`
-.. Comment out the Solr Client Configuration for *Cloud Solr Client* and *Embedded Solr Client* sections.
-.. Uncomment the section for the *Http Solr Client*.
-
-[NOTE]
-====
-To point to an External Solr server instead of the default Internal one, put the url in `solr.http.url` (see below for example).
-Also, two jar files need to be added to the Solr installation as described in <<Installing Solr Cloud,Installing Solr Cloud>>.
-====
+. Change the property `solr.http.url` to read `https` instead of `http`
 
-.system.properties
-----
-
-solr.client = HttpSolrClient
-solr.http.url = https://solr-host-name:8993/solr
-solr.data.dir = ${karaf.home}/data/solr
-
-----
+* Change Solr's TCP port
+. Edit the `${home_directory}/etc/system.properties`
+. Change the property `solr.http.port` to the desired value
 
-.[[_http_solr]]Embedded Solr
-The Embedded Solr is an embedded, local Apache Solr server instance.
-It is a local instance that is a lightweight Catalog Provider solution.
-However, it does not provide a Solr Admin UI or a REST-like HTTP/XML and JSON API.
-If that is necessary, see <<HTTP Solr,HTTP Solr>> or <<Solr Cloud,Solr Cloud>>.
+.Internally Managed Solr Server System Properties
+ ----
+ solr.client=HttpSolrClient
+ solr.http.port=8994
+ solr.http.url=http://localhost:${solr.http.port}/solr
+ solr.managed.internally=true
+ ----
 
-.Embedded Solr Deployment
-[ditaa,embedded-solr-diagram,png]
-....
- Container (JVM)
- +=-----------------+
- |                  |
- | /--------------\ |
- | |cDEF          | |
- | | DDF          | |
- | |              | |
- | |  +---------+ | |
- | |  |Solr cFF0| | |
- | |  +---------+ | |
- | |              | |
- | \--------------/ |
- |                  |
- |                  |
- +------------------+
-....
+Externally Managed Solr Server:: In this configuration, ${branding} communicates with a
+Solr server that is started, stopped, and otherwise administrated independently of ${branding}.
+Some considersations of this configuration are:
 
-.When to Use the Embedded Solr
-The Embedded Solr configuration is great for demonstrations, training exercises, or for sparse querying and ingesting. See below for details.
+* Flexible. The externally managed Solr server can be located on any host. ${branding} uses
```

Comment:
Same issue with Flexible here as in the Enable TLS comment.

Reply:
It seems to read well without the word "Flexible", so I took it out.